package.json

```diff
@@ -30,10 +30,18 @@
     "typescript": "3.3.1"
   },
   "devDependencies": {
+    "@commitlint/cli": "7.5.2",
+    "@commitlint/travis-cli": "7.5.2",
```

Comment:
I don't think we should be including this here since we don't use Travis CI for everything we do. We should add it to our generator, but we already have [an issue for that](https://github.com/silvermine/generator-typescript/issues/33) which would include the [changes we made in Lambda Express](https://github.com/silvermine/lambda-express/pull/15), so would add it there when the user chooses to generate open source project.

Reply:
I added this here to run commitlint for this repo. It's in devDependencies so it shouldn't affect the consuming repo